### ISSUE TITLE
TF530 in Atari ST computers

### ISSUE THREAD

**Issue by MrPetari** [terriblefire/tf530#13]
Sorry for this, but I can not contact you, Stephen Leary other way. Don't see any e-mail address, and no way I will PM you at exxos forum.
This IS very related to this board:
So, why I left exxos forum ? No not banned, I was banned there after I publicly announced it and gave reasons in short, in now deleted (of course, they always delete such threads)  .
Look this fresh post:
<a href ="https://www.exxoshost.co.uk/forum/viewtopic.php?f=67&t=1385&sid=eb661e4309193be472f39454677efbc6&start=10">TF thread link</a><br>
That's what I suggested in that thread, some 3 weeks ago. + I offered my TOS 1.04 mod, with large stackframe support there - so that TF530 can work not only with TOS 2.06 .  And what happened ?  His greatness exxos deleted my 4 posts there not saying me a word about. Even after I did that modded 1.04 for him, for his accelerators. Yes, sounds absurd, but that's what happened.
In PM he said that my posts made there confusion. No, only in his head. I'm really worried about his mental health. Without any offense, insult - there are signs of mental degradation. Or he is just too tired, or his ego became too big, and that goes with shallowness.

Conclusion: now TF does what I suggested first, my posts are deleted, and now it is presented as it is exxos's idea ?
Does it look for you as theft ? "

Just to add that I suggested in deleted posts to add on board self TOS 2.06 address decoding logic for ST computers (in STE no need for). I talked about my TOS 1,04 mod with large stackframe support. And then booom - all it was deleted.   
Yes, this is issue, even if not with board self, but in some way it is related with board, and it's future.

**Comment by MrPetari** [comment]
Yes, as expected - usual ignorance. But that will not make things better and your bad behavior forgotten.
There is people with good memory, with balls. We will use every opportunity to say what really happened. You can ask admins to ban us, we will find another place, ways to tell the truth. And that's because we care for common good, not for own narrow interest. Have a fun with your Don Corleone, exxos .

**Comment by terriblefire** [comment]
The TF530 cannot decode properly for the Atari ST. If you had half a brain and looked at the schematic you can see this. GLUE needs to decode address 0 and address 4 for the ROM during startup. The TF530 does not have all the address lines needed to do this. You accuse me of ignorance?? Please.. 

I tried this before and it didnt work. This cannot work on the TF530. Instead of being stupid like you i created a new rom board for the ST.

https://github.com/terriblefire/tfboard64

Closing the issue because the new board fixes the issue. 

If you want to try and make the TF530 work you are welcome to. The sources are open. All the tools are free. If you are smart you can do it.... oh wait. :)

**Comment by terriblefire** [comment]
> Yes, as expected - usual ignorance. But that will not make things better and your bad behavior forgotten.
> There is people with good memory, with balls. We will use every opportunity to say what really happened. You can ask admins to ban us, we will find another place, ways to tell the truth. And that's because we care for common good, not for own narrow interest. Have a fun with your Don Corleone, exxos .

**Comment by terriblefire** [comment]
I've also reported you for abuse to github.

**Comment by MrPetari** [comment]
Sure, only thing I want in my life is to test another people's designs. 
I see what you communicating with exxos at his forum. That's something I would never do. Exchanging 15 posts about something what can be resolved in 5 minutes. Engaging army of testers, because can not design even simpler PCB without errors. You joined a genius :-)
We could make some cooperation, but you acted arrogant, exxos even worse. 
I designed lot of HW myself, many of it before Internet era. But you don't care.  
Exxos forum is disgrace, he made IDE adapter design, and now it appears that he even did not understand how IDE protocol works. And needs testers, of course. But it will need much more than testing - redesign. I did IDE adapters in 1992 - for Atari ST and for Sinclair Spectrum. Later CF card adapters. All it worked and works fine. I did not need anyone else to test it for me.
Thank you for being AH .
